### ISSUE TITLE
core: create documentation for c# developers

### ISSUE THREAD

**Issue by ngosang** [Jackett/Jackett#10048]
We already have a nice documentation for Cardigann indexers. => https://github.com/Jackett/Jackett/wiki/Definition-format
The task is to create a similar documentation doc c# indexers.

**Comment by xfouloux** [comment]
Indeed, with a little bit of documentation, i think i would be able to tackle some C# implementation for some trackers.
